configs/12.0/packages/valgrind/sources

```diff
@@ -22,6 +22,7 @@
 ATSRC_PACKAGE_NAME="Valgrind"
 ATSRC_PACKAGE_VER=3.14.0
 ATSRC_PACKAGE_REV=0c701ba2a4b1
+ATSRC_PACKAGE_BRANCH=VALGRIND_3_14_BRANCH
```

Comment:
Is this correct? git can't seem to find this commit on that branch:
```
$ git branch -r --contains 0c701ba2a4b1
  origin/HEAD -> origin/master
  origin/VALGRIND_3_16_BRANCH
  origin/grail
  origin/master
```

Reply:
It should be `3a3000290`, head of VALGRIND_3_14_BRANCH.

Reply:
Ah, that's unfortunate. Better not create a new problem then, we can leave it as-is.